packages/lit-dev-content/site/tutorials/content/working-with-lists/05.md

```diff
@@ -0,0 +1,104 @@
+When using the `map()` directive or just providing an iterable within the
+template, Lit creates a list of DOM nodes and updates them in the iteration
+order, efficiently reusing any previously rendered nodes. This is **good** and
+should be utilized when **all states are being controlled by Lit**.
```

Comment:
How about: `These techniques should only be used when Lit controls all of the rendered state as is commonly the case.`